packages/devtools_app/integration_test/test/live_connection/debugger_panel_test.dart

```diff
@@ -59,27 +60,44 @@ void main() {
       isTrue,
     );
 
-    logStatus('opening the "more" menu');
+    logStatus('Navigating to line 57...');
 
-    final moreMenuFinder = find.byType(PopupMenuButton<ScriptPopupMenuOption>);
-    expect(moreMenuFinder, findsOneWidget);
-    await tester.tap(moreMenuFinder);
-    await tester.pumpAndSettle(safePumpDuration);
+    await goToLine(tester, lineNumber: 57);
 
-    logStatus('selecting the go-to-line menu option');
+    logStatus('looking for line 57');
 
-    final goToLineOptionFinder = find.textContaining('Go to line number');
-    expect(goToLineOptionFinder, findsOneWidget);
-    await tester.tap(goToLineOptionFinder);
-    await tester.pumpAndSettle(safePumpDuration);
+    // Look for the line 57 gutter item:
+    final gutter57Finder = findGutterItemWithText('57');
+    expect(gutter57Finder, findsOneWidget);
 
-    logStatus('entering line number in the go-to-line dialog');
+    // Look for the line 57 line item:
+    final line57Finder = findLineItemWithText("print('Hello!');");
+    expect(line57Finder, findsOneWidget);
 
-    final goToLineInputFinder = find.widgetWithText(TextField, 'Line Number');
-    expect(goToLineInputFinder, findsOneWidget);
-    await tester.enterText(goToLineInputFinder, '30');
-    await tester.testTextInput.receiveAction(TextInputAction.done);
-    await tester.pumpAndSettle(safePumpDuration);
+    // Verify that the gutter item and line item are aligned:
+    expect(
+      areHorizontallyAligned(
+        gutter57Finder,
+        line57Finder,
+        tester: tester,
+      ),
+      isTrue,
+    );
+
+    logStatus('setting a breakpoint');
+
+    // Tap on the gutter for the line to set a breakpoint:
+    await tester.tap(gutter57Finder);
+    await tester.pumpAndSettle(longPumpDuration);
+
+    logStatus('performing a hot restart');
+
+    await tester.tap(find.byKey(const Key('Hot Restart Button')));
```

Comment:
instead of adding a Key to hot restart button for testing purposes, just find the button by its type `find.byType(HotRestartButton)`

Reply:
Ah nice, done!